packages/salesforcedx-vscode-core/test/vscode-integration/orgPicker/orgList.test.ts

```diff
@@ -330,6 +330,18 @@ describe('Set Default Org', () => {
     ).to.be.true;
   });
 
+  it('should return Continue and call force:org:list:clean command if SFDX: Remove Deleted and Expired Orgs is selected', async () => {
+    orgListStub.returns(orgsList);
```

Comment:
I know this matches the other tests, but this doesn't appear to actually test the code.  When it runs, the ```'$(plus) ' +
        nls.localize('force_auth_web_login_authorize_dev_hub_text')``` case gets hit, not the  nls.localize('force_org_list_clean_text') case.

Reply:
I spoke with Randi about this, and this isn't an issue.  This case is supposed to hit the default case, so all is good.